corehq/apps/app_manager/static/app_manager/js/app_manager.js

```diff
@@ -242,7 +242,7 @@
 
         COMMCAREHQ.app_manager.commcareVersion.subscribe(function () {
             $('.commcare-feature').each(function () {
-                var version = '' + $(this).data('since-version') || '1.1',
+                var version = '' + $(this).attr('data-since-version') || '1.1',
```

Comment:
Not sure why you changed this. If it wasn't working perhaps its because we've upgraded JQuery to > 1.6 and need to instead query for `sinceVersion`: http://api.jquery.com/data/#data-html5